vespa-chinese-linguistics/src/main/java/com/qihoo/language/JiebaTokenizer.java

```diff
@@ -0,0 +1,105 @@
+// Copyright 2017 Yahoo Holdings. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
```

Comment:
Please change to your copyright: Copyright 2020 Qihoo Corporation.

Reply:
Changed.